introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+### Node Types
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes
+are not aware of cluster-wide distribution policies, and otherwise look just as if they were
```

Comment:
Not sure what "cluster-wide distribution policies" are.

Reply:
Maybe I'll just use the same verbage in the section above. Something like: Data nodes do not store cluster-wide information, and otherwise look just as if they were single TimescaleDB instances.